code/drasil-code/lib/Language/Drasil/Code/Imperative/ReadInput.hs

```diff
@@ -7,7 +7,7 @@ import Language.Drasil hiding (Data, Matrix, CodeVarChunk)
 import Language.Drasil.Code.DataDesc (DataDesc'(..), Data'(..), DataItem'(..),
   Delimiter, dataDesc, junk, list, singleton')
 import Language.Drasil.Chunk.Code (CodeVarChunk)
-import Language.Drasil.Expr.Development (Expr(Matrix))
+import Language.Drasil.Expr.Development (Expr(Matrix, Clif)) -- TODO: remove Matrix entirely
```

Comment:
Ah, if I had not read the code sequentially, I would have seen this!

Reply:
Yes, as above, I would like some thoughts on what to replace it with.